packages/webcert/src/feature/certificate/Certificate.tsx

```diff
@@ -36,12 +36,11 @@ const Wrapper = styled.div`
   .contentPaperWrapper {
     padding-left: 32px;
     padding-right: 32px;
+    padding-bottom: 20px;
   }
 `
 
 const CategoryWrapper = styled.div`
-  background: #ffffff;
```

Comment:
We can't remove the background from the CategoryWrapper without affecting the displayed validationerrors in category  funktionsnedsattningar:

![image](https://user-images.githubusercontent.com/57347249/214057622-8119cc88-3699-4a38-8677-28cf30ac12e9.png)